<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieInfoDialog</class>
 <widget class="QDialog" name="MovieInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie Information</string>
  </property>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="releaseDateLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Release Date:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlotLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RatingLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>65</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DirectorsLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Directors:</string>
   </property>
  </widget>
  <widget class="QLabel" name="GenresLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Genres:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MovieName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MovieRating">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MovieReleaseDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MovieDirectors">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MovieGenres">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MoviePlot">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="MovieCast">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MovieCastLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Movie Cast:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="EpisodesList">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EpisodesLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Episodes:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewEpisodeButton">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>542</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View Episode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
